src/components/Annotation/index.tsx

```diff
@@ -13,14 +12,14 @@ export interface Props {
   id: ItemId;
 }
 
-const Annotation: React.FunctionComponent<Props> = ({
+const Annotation: React.FC<Props> = ({
   data,
   xScale,
   height,
   color = '#e8336d',
   fillOpacity = 0.1,
   id,
-}) => (
+}: Props) => (
```

Comment:
We shouldn't need `: Props` here

Reply:
Unfortunately, eslint complains about prop validation since we have the `react/prop-types` rule on. Similar situation to insight

Reply:
Argh.